jsapp/js/components/processing/sidebar/sidebar.tsx

```diff
@@ -0,0 +1,51 @@
+import React, {useState} from 'react';
+import singleProcessingStore, {
+  StaticDisplays,
+} from 'js/components/processing/singleProcessingStore';
+import TransxDisplay from 'js/components/processing/transxDisplay';
+import SidebarDisplaySettings from 'js/components/processing/sidebar/sidebarDisplaySettings';
+import type {AssetResponse} from 'jsapp/js/dataInterface';
+import SingleProcessingSubmissionData from 'js/components/processing/sidebar/sidebarSubmissionData';
+import SingleProcessingSubmissionMedia from 'js/components/processing/sidebar/sidebarSubmissionMedia';
```

Comment:
Both of these imports have different internal names, let's use them to avoid confusion. I.e. there is no `SingleProcessingSubmissionData` to be found, but there is a `SidebarSubmissionData`